### PR TITLE
Contact me form for my portfolio

### PR DESCRIPTION
**Pull request by D4nielJ**
## Hello Reviewer

During these commits, my coding partner and I, focus on the Contact Me form at the end of the portfolio. Only for mobile sizes for now.

Contact-me Form has the right validation: 

- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.

Contact-me form collects data using Formspree Service.

You can check the [Live Demo here.](https://d4nielj.github.io/developer-portfolio/)

Let me know any changes or suggestions.

Cheers,
Daniel Jaramillo

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -192,5 +191,33 @@ <h2>Skills</h2>
...
+            I’d love to help with it</p>
+            <ul>
+                <li>
+                    <input type="text" class="form-input-box" name="user_name" placeholder="Enter your name here" required maxlength="30">
+                </li>
+                <li>
+                    <input type="email" class="form-input-box" name="user_email" placeholder="Enter your e-mail here" required>
+                </li>
+                <li>
+                    <textarea class="form-input-box form-input-box-area" placeholder="Write your message here" name="user_message" required maxlength="500"></textarea>
```
Comment (Davidosky007) [inline index.html:214]: I suggest message input's type equals textarea as stated in the project requirement

File: index.html [hunk 2]
```diff
@@ -192,5 +191,33 @@ <h2>Skills</h2>
...
     </section>
+
+    <!--contact-me form section-->
+    <section id="contact-me">
+      <form
+        class="contact-me-form"
+        action="https://formspree.io/f/mjvjrbzg"
+        method="post"
+      >
+        <h1>Contact me</h1>
```
Comment (Davidosky007) [inline index.html]: This is a subtitle i suggest you wrap it an `hx` tag and not `h1` tag. `h1` tag is used for main title get more information [Here](https://www.w3schools.com/tags/tag_hn.asp)

File: css/styles.css [hunk 3]
```diff
@@ -334,6 +334,72 @@ header {
...
 }
 
+.contact-me-form {
+  min-height: 90vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 50px 5%;
+  background: #6070ff url(../media/contact-form-shapes-mobile.svg) 135% 50% no-repeat;
```
Comment (Davidosky007) [inline css/styles.css]: You can make the background-position same as the design template this is yours 
Screenshot (25)

This is the design template please compare and make necessary adjustment 👍 
Screenshot (24)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @the-catalystmc @D4nielJ, 

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest you start your commit message with capital letters
* I suggest your Pull Request should have a short summary describing introduced changes in more detail. please make it more detailed as stated in the project requirement 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (D4nielJ) [reply index.html:214]: `textarea` doesn't allow for `type="textarea"`, I think it's already implied by the tag. I tried doing it like you said. A Linter error came up.

File: index.html (hunk 2 above)
Comment (D4nielJ) [reply index.html]: Done

File: css/styles.css (hunk 3 above)
Comment (D4nielJ) [reply css/styles.css]: Done

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html:214]: ok you can undo it am sorry its a mistake on our end 🙏 kindly fix the other changes and submit them for review 👍

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @D4nielJ @the-catalystmc!

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

shine team

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**